### PR TITLE
ESPnet2 Receipe for swbd

### PR DESCRIPTION
**Pull request by yuekaizhang**
Current result:
ESPnet 1 6.8/14.0
ESPnet 2 8.4/15.6  (still need to tune it and run more epochs)

### CODE REVIEW

File: egs2/TEMPLATE/asr1/asr.sh [hunk 1]
```diff
@@ -1281,6 +1281,7 @@ if ! "${skip_eval}"; then
         done
 
         [ -f local/score.sh ] && local/score.sh ${local_score_opts} "${asr_exp}"
+        [ -f local/score_swbd.sh ] && local/score_swbd.sh ${_data} ${_dir}/score_wer
```
Comment (kamo-naoyuki) [inline egs2/TEMPLATE/asr1/asr.sh:1284]: Why don't you create local/score.sh? Please not modify asr.sh as possible to invoke local scritps.

Comment (yuekaizhang) [reply egs2/TEMPLATE/asr1/asr.sh]: Sure, I have fixed it now.

Comment (kamo-naoyuki) [reply egs2/TEMPLATE/asr1/asr.sh]: Thank you!

File: egs2/swbd/asr1/RESULTS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- espnet version: `espnet 0.9.9`
+- pytorch version: `pytorch 1.8.0+cu111`
+- Git hash: `64f026d35013e9f0058bcdeab86eb28fed48ed4b`
+  - Commit date: `Fri May 7 09:31:16 2021 +0000`
+
+## asr_train_asr_cformer5_raw_bpe2000_sp
+### WER
+
+```
+exp_sp/train_nodup_sp_pytorch_train_pytorch_conformer_lr5_specaug_resume/decode_eval2000_model.last10.avg.best_decode_train_transformer_lm_pytorch_swbd+fisher_bpe2000/scoring/hyp.callhm.ctm.filt.sys
```
Comment (sw005320) [inline egs2/swbd/asr1/RESULTS.md:16]: Could you double-check that the number of reference words is exactly the same as espnet1 or Kaldi results?

Comment (sw005320) [reply egs2/swbd/asr1/RESULTS.md:16]: I have checked it and it's correct.

**Comment by roshansh-cmu** [comment]
I was able to get 7.2/14.6 without LM Rescoring using this config with conformer:

[swbd_conformer.txt](https://github.com/espnet/espnet/files/6624836/swbd_conformer.txt)

Happy to share models if needed

Still tuning LM weights for fusion

**Comment by sw005320** [comment]
That's very good, @roshansh-cmu.
Year, after we merge this PR, I want to ask you to upload your config and model.

File: egs2/swbd/asr1/RESULTS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri May 14 07:43:22 UTC 2021`
+- model link: https://drive.google.com/drive/folders/18jJyFxSS9nhS51H8xWkMiDybGKrKPVEV?usp=sharing
```
Comment (sw005320) [inline egs2/swbd/asr1/RESULTS.md:5]: Can you provide a model through zenodo?
Please check https://github.com/espnet/espnet_model_zoo#espnet-model-zoo

**Review by sw005320 — APPROVED** [review]
Can you upload a model via model zoo?

File: egs2/swbd/asr1/RESULTS.md (hunk 3 above)
Comment (yuekaizhang) [reply egs2/swbd/asr1/RESULTS.md:5]: Sorry, I just saw it. I would upload it through model-zoo.

**Comment by sw005320** [comment]
Thanks a lot!

**Pull request merged**